src/Backoffice/Employee/Domain/DataTransferObjects/EmployeeDto.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\Domain\DataTransferObjects;
+
+class EmployeeDto
+{
+    public function __construct(
+        private readonly string $name,
+        private readonly string $email,
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function getEmail(): string
+    {
+        return $this->email;
+    }
```

Comment:
```suggestion
    public function __construct(
        public readonly string $name,
        public readonly string $email,
    ) {
    }
```
Marking an attribute as read-only prevents violating the encapsulation principle. This way, you may access attributes directly without the need of a getter.